kythe/typescript/testdata/function.ts

```diff
@@ -84,3 +86,14 @@ function bindingTest({aa, bb: {cc, dd}}, [ee, [ff]]: NestedArray) {
   //- @ff ref FF
   dd = ff;
 }
+
+// Test function expressions introducing an anonymous scope
+//- @ev defines/binding Ev
+//- @an defines/binding An
+let ev, an;
+(function() {
+//- !{ @ev ref Ev }
+let ev;
```

Comment:
This is defines/binding, not ref, so I think it's always true even without your change.

Reply:
Fixed.